src/App.js

```diff
@@ -7,8 +7,18 @@ const App = () => {
 
   const handleGamePlay = (index) => {
     let updatedBoard = [...squares]
-    updatedBoard[index] = "X"
-    setSquares(updatedBoard)
+    if (counter % 2 === 0) {
+      updatedBoard[index] = "X"
+      setSquares(updatedBoard)
+    } else {
+      updatedBoard[index] = "O"
+      setSquares(updatedBoard)
+    }
+  }
+
+  const [counter, setCounter] = useState(2)
```

Comment:
Cool approach to change user!  Is there a reason why you are setting state to 2 initially?

Reply:
since we're using the modulo method, i wanted to avoid any messiness that can come about from 0 and 1. Seemed an easy solution to just start it from 2